src/comp/comp.py

```diff
@@ -20,52 +20,54 @@ def __repr__(self):
     Human("Igon", 41),
     Human("David", 31),
 ]
+print(type(len(humans)))
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [person.name for person in humans if person.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [person.name for person in humans if person.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
+
 print("Starts between C and G, inclusive:")
-c = []
+c = [person.name for person in humans if person.name[0] == "C" or person.name[0] == "D" or person.name[0] == "E" or person.name[0] == "F" or person.name[0] == "G"]
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = [person.age + 10 for person in humans ]
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [f"{person.name}-{person.age}" for person in humans ]
 print(e)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
-f = []
+f = [(person.name, person.age) for person in humans if 26 < person.age < 33]
 print(f)
 
 # Write a list comprehension that creates a list of new Humans like the old
 # list, except with all the names uppercase and the ages with 5 added to them.
 # The "humans" list should be unmodified.
 print("All names uppercase:")
-g = []
+g = [(person.name.upper(), person.age + 5) for person in humans]
```

Comment:
It's a list comprehension that creates a list of new Humans so you need to pass the values to an instance of the Human class like this Human(value1, value2)